cmd/guacingest/cmd/ingest.go

```diff
@@ -95,7 +95,7 @@ func ingest(cmd *cobra.Command, args []string) {
 
 	emit := func(d *processor.Document) error {
 		if err := ingestor.Ingest(ctx, d, opts.graphqlEndpoint, transport, csubClient); err != nil {
-			logger.Errorf("unable to ingest document %q : %v", d.SourceInformation.Source, err)
+			return fmt.Errorf("unable to ingest document %q : %v", d.SourceInformation.Source, err)
```

Comment:
This undoes the change in https://github.com/guacsec/guac/pull/1832 where we want to ack and log an error if there is an error in parsing the document so that we don't leave a bad document in the pubsub to be re-ingested forever.

Reply:
hmm, should we add a health check for the graphQL server to see if it is down and use the error to determine that it is a "recoverable error".

Reply:
fixed!